src/com/company/exercises/FIs.java

```diff
@@ -0,0 +1,46 @@
+package com.company.exercises;
+
+import java.util.function.Consumer;
+import java.util.function.Function;
+import java.util.function.Predicate;
+import java.util.function.Supplier;
+
+public class FIs {
+
+    static class Person {
+        String name;
+        int age;
+
+        public Person(String name, int age) {
+            this.name = name;
+            this.age = age;
+        }
+    }
+
+    public static void main(String[] args) {
+//        Supplier
+        Supplier<Integer> supplier = () -> 42;
+        System.out.println(supplier.get());
+
+//        Function
+        Function<Integer,String> function = age -> age > 21 ? "you can drink" : "sober";
```

Comment:
I am probably nitpicking now, but try to use Ctrl + Alt + L to reformat your code before submitting it to git.

Reply:
Done.